test/helper.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
-load /usr/local/bats/support/load.bash
-load /usr/local/bats/assert/load.bash
-load /usr/local/bats/file/load.bash
+load ../bats/lib/support/load.bash
```

Comment:
Maybe add an `if [[ ! -d ../bats/lib/support ]]; then printf "bats libraries missing, run get_bats_lib.sh "; exit 1; fi` here. Then people don't have to consult the documentation when a loading error occurs.

Reply:
That turns out to be a bit awkward to do.

The path given to the `load` function is relative to the test being run, whereas the path tested for in such a statement is relative to `$PWD`. Which makes the code look odd because there will be a mismatch between the path tested and the one loaded. Are you sure you want that?